apps/web/src/features/spaces/components/SpaceCard/SpaceContextMenu.tsx

```diff
@@ -57,12 +59,14 @@ const SpaceContextMenu = ({ space }: { space: GetSpaceResponse }) => {
           <ListItemText>Rename</ListItemText>
         </MenuItem>
 
-        <MenuItem onClick={(e) => handleOpenModal(e, ModalType.REMOVE)}>
-          <ListItemIcon>
-            <SvgIcon component={DeleteIcon} inheritViewBox fontSize="small" color="error" />
-          </ListItemIcon>
-          <ListItemText>Remove</ListItemText>
-        </MenuItem>
+        <Track {...SPACE_EVENTS.DELETE_SPACE_MODAL} label={SPACE_LABELS.space_dashboard_card}>
```

Comment:
I would suggest adding a new label here like `space_context_menu` because this is also used on the space list page and not just the dashboard/sidebar